crates/ironrdp-cliprdr/src/pdu/format_data/mod.rs

```diff
@@ -166,6 +166,10 @@ impl<'a> FormatDataResponse<'a> {
     pub fn into_owned(self) -> OwnedFormatDataResponse {
         self.into_owned_pdu()
     }
+
+    pub fn into_vec(self) -> Vec<u8> {
```

Comment:
suggestion: Return Cow<'a, [u8]> and leave the choice to the caller

Reply:
ok